proof/crefine/ARM/Wellformed_C.thy

```diff
@@ -1,4 +1,5 @@
 (*
+ * Copyright 2023, Proofcraft Pty Ltd
```

Comment:
I have a concern on these (`Wellformed_C`): can we claim copyright if we only remove a bunch of lemmas from a file? @lsf37 ?

Reply:
Let's leave it out if we end up just removing lines.

Reply:
No problem, happy to get rid of it. How about all of the other files, where it's largely just removing rules passed to `simp` and restyling?